Dockerfile

```diff
@@ -1,29 +1,39 @@
-FROM ubuntu:bionic
+FROM php:7.3-apache
 
-# TODO: think through optimizations here once this is working.
-RUN    apt-get update \
-    && apt --yes install software-properties-common \
-    && apt-add-repository --yes --update ppa:ansible/ansible \
-    && apt --yes install git ansible
+# Extensions we know LF requires/desires:
+# curl - already provided in image
+# mbstring - already provided in image
+# xdebug
+# intl
+# gd
+# mongodb
+#
+# see https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions gd xdebug mongodb curl intl
```

Comment:
it would be premature at this point but let's add a TODO: in here so we remember to come back and optimize these "layers", i.e., we'll want to do things like this in one line once we get past the "make it work" phase and begin optimization and housekeeping.